Src/AutoFixture.NUnit3.UnitTest/ThrowingStubFixture.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    /// <summary>
+    /// A <see cref="IFixture"/> for the benefit of creating stubs of <see cref="AutoDataAttribute"/> for unit testing
+    /// </summary>
+    public class ThrowingStubFixture : IFixture
```

Comment:
I think the original one being named **ThrowingStubFixture ** actually makes sense as it describes its purpose. However for **InlineAutoData** I am not really sure. Maybe I can just create an inline **Stub **as it doesn't really do anything?